migration/migrator/migrations/course/20200728162003_add_seek_message.py

```diff
@@ -0,0 +1,25 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
```

Comment:
Please add these import statements
```suggestion
"""Migration for a given Submitty course database."""
import json
from pathlib import Path
def up(config, database, semester, course):
```